nose_launchable/client.py

```diff
@@ -78,12 +78,11 @@ def start(self, build_number):
         response_body = res.json()
         logger.debug("Response body: {}".format(response_body))
 
-        self.test_session_id = response_body['id']
+        self.test_session_id = "builds/{}/test_sessions/{}".format(
```

Comment:
`self.test_session_id` is used to construct urls for uploading test case results and closing test sessions so I don't think changing `self.test_session_id` itself is a good idea. Instead, I believe you should somehow pass build_name (or build_number) to the subset method. 


https://github.com/launchableinc/nose-launchable/blob/60f0192c30c9290bd937ae501735a38131a3e327/nose_launchable/client.py#L113-L119

https://github.com/launchableinc/nose-launchable/blob/60f0192c30c9290bd937ae501735a38131a3e327/nose_launchable/client.py#L128-L135

Reply:
Hmm, How about change to use fixed test_session_id (buils/<build number>/test_sessions/<id> format) in uploading test case results and closing test session?
Because I think we should gather a logic that building test session id in one place.

Reply:
That could work, but then the name `self.test_session_id` looks strange to me since it is not. I don't want to step into how to implement it but If I were you, I would create a new small class like `TestSessionContext` and store build id and test session id to its instance. Then we can call getTestSessionId or getBuildPath to extract the necessary information.

Reply:
Hmm, I'm also wondering why those unite tests did not fail...
https://github.com/launchableinc/nose-launchable/blob/83beb539d1365178a4d5ee6462368b182eb3f77e/tests/test_client.py#L217

Reply:
I think set test_session_id directly so this test didn't fail 
https://github.com/launchableinc/nose-launchable/blob/83beb539d1365178a4d5ee6462368b182eb3f77e/tests/test_client.py#L204-L205